buildscripts/azure/azure-linux-macos.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  name: ''
+  vmImage: ''
+  matrix: []
+
+jobs:
+- job: ${{ parameters.name }}
+  pool: 
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    maxParallel: 11
+    matrix:
+      ${{ insert }}: ${{ parameters.matrix }}
+
+  steps:
+    - script: |
+        # Sleep 10 seconds because of race condition with background apt process
```

Comment:
this comment can be removed since we already deleted the sleep (we should go fix this in Numba as well)